Barotrauma/BarotraumaClient/Source/Characters/CharacterHUD.cs

```diff
@@ -195,9 +201,37 @@ public static void Draw(SpriteBatch spriteBatch, Character character, Camera cam
                         };
                     }
 
+                    if (grabHoldButton == null)
+                    {
+                        grabHoldButton = new GUIButton(
+                            new Rectangle(character.SelectedCharacter.Inventory.SlotPositions[0].ToPoint() + new Point(320, -60), new Point(130, 20)),
+                                "Grabbing: " + (character.AnimController.GrabLimb == LimbType.Torso ? "Torso" : "Hands"), "");
```

Comment:
This isn't very future-proof for other types of grab points

Reply:
Yeah, true. I'll look into it later on.